.github/workflows/label_docs.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  dispatch_create:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'labeled' && github.event.label.name == 'doc needed'
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create issue if doc needed label is applied
+        uses: actions/github-script@v5    
+        with:
+          github-token: ${{ secrets.PAT_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'noir-lang',
+              repo: 'docs',
+              workflow_id: 'new-migrated-issue.yml',
+              ref: 'main',
+              inputs: {
+                name: "${{ github.event.pull_request.number }}"
+              }
+            });
```

Comment:
We already have an established way to trigger reusable workflows which we use for triggering releases:

https://github.com/noir-lang/noir/blob/73df4653556a7d1c74d184e27ec5a8ca3be47af9/.github/workflows/release.yml#L66-L73

Could you update to use this?